Lib/typing.py

```diff
@@ -433,6 +433,9 @@ def __reduce__(self):
     def __call__(self, *args, **kwds):
         raise TypeError(f"Cannot instantiate {self!r}")
 
+    def __iter__(self):
```

Comment:
This is a little sad, because now Union is duck type-compatible with `collections.abc.Iterable`. @serhiy-storchaka recently suggested setting `__iter__ = None` for cases like this.

Reply:
It is not I suggested recently. Treating `__iter__ = None` specially was implemented many years ago.

Reply:
Well, you brought up the general pattern :)

Reply:
Oh, nice. Switched to `__iter__ = None`.